kolibri/plugins/learn/assets/src/views/LibraryPage/UnPinnedDevices.vue

```diff
@@ -1,24 +1,34 @@
 <template>
 
-  <div
-    class="card-main-wrapper"
-    :style="cardStyle"
-  >
-    <div class="" style="width:250px;height:100px;">
-      <h2 class="device-name">
-        <span>
-          <KIcon icon="device" />
-        </span>
-        <span>
-          <TextTruncator
-            :text="deviceName"
-            :maxHeight="52"
-          />
-        </span>
-      </h2>
-      <p class="channels">
-        {{ $tr('channels', { count: channels }) }}
-      </p>
+  <div>
+    <div
+      class="card-main-wrapper"
+      :style="cardStyle"
+    >
+
+      <KRouterLink
+        v-if="allDevices !== null"
+        :text="allDevices.nickname.length ? allDevices.nickname : allDevices.device_name"
+        :to="{ name: 'LIBRARY', params: { deviceId: allDevices.id } }"
+        style="text-decoration:none;width:100%;"
+      >
+        <h2 class="device-name">
+          <span>
+            <KIcon :icon="getDeviceIcon()" />
+          </span>
+          <span>
+            <TextTruncator
+              :text="deviceName"
+              :maxHeight="52"
+            />
+          </span>
+        </h2>
+        <div class="channels">
+          <p v-if="channels">
+            {{ channels }} {{ $tr('channels', { count: channels }) }}
+          </p>
+        </div>
```

Comment:
This is another place where the number of `div`s can be reduced. 

The spacing and alignment of cards is tricky to get right. Here is how I thought about this card (please excuse my terrible penmanship): 

<img width="747" alt="Screenshot 2023-03-31 at 1 32 46 PM" src="https://user-images.githubusercontent.com/17235236/229190065-424ea8e5-6e4e-4064-a699-8e59f4c62ec0.png">
<img width="500" alt="Screenshot 2023-03-31 at 1 05 40 PM" src="https://user-images.githubusercontent.com/17235236/229190283-b88e70ed-62c9-42f6-8aca-7ccdf5d5d67b.png">

- First, you can set a specific height on your card, based on the spec (80px)
- The next thing I looked for was how the card was divided up. I see a few different sections: 
1. The left part of the card has the icon in it, only. 
2. The other piece of the card has text, and the text is left-aligned 
3. Using the text truncation is a great idea 👍 
4. I think using an absolute position for the "Channels" text would make the cards look cleaner. By this I mean, the channels text should always appear in the same bottom part of the card, whether the title portion is one or two lines. Using absolute positioning can help with this.